seed_data/management/commands/alter_data.py

```diff
@@ -0,0 +1,320 @@
+def set_course_to_enrolled(user=None, course_run=None, in_future=False,  # pylint: disable=too-many-arguments
+                           grade=None, audit=False, missed_deadline=False):
+    """Sets a course to have a currently-enrolled course run"""
+    enrollable_setting = dict(enrollable_past=True) if missed_deadline else dict(enrollable_now=True)
+    if in_future:
+        set_course_run_future(course_run, enrollable_now=True, save=True, **enrollable_setting)
+    else:
+        set_course_run_current(course_run, upgradeable=not missed_deadline, save=True, **enrollable_setting)
+    CachedEnrollmentHandler(user).set_or_create(course_run, verified=not audit)
+    if grade:
+        CachedCurrentGradeHandler(user).set_or_create(course_run, grade=grade)
+    return course_run
+
+
+@course_state_editor
+def set_course_to_needs_upgrade(**kwargs):
+    """Sets a course to have an enrolled course run that needs to be upgraded (aka 'audit')"""
+    kwargs.update(dict(audit=True, in_future=False))
+    return set_course_to_enrolled(**kwargs)
+
+
+@accepts_or_calculates_now
+def add_future_run(user=None, course=None, now=None):
+    """Adds a future enrollable course run for a course"""
+    base_edx_course_key = '{}-{}'.format(NEW_COURSE_RUN_PREFIX, now.strftime('%m-%d-%Y'))
+    default_title = base_edx_course_key
+    key_append = 0
+    new_course_run = None
+    created = False
+    with transaction.atomic():
+        while not new_course_run or not created:
+            new_course_run, created = CourseRun.objects.get_or_create(
+                course=course,
+                title=default_title,
+                edx_course_key='{}-{}'.format(base_edx_course_key, key_append),
+            )
+            key_append += 1
+    set_course_run_future(new_course_run, save=True)
+    CachedEnrollmentHandler(user).set_or_create(course_run=new_course_run, blank=True)
+    return new_course_run
+
+
+def clear_future_runs(user=None, course=None):  # pylint: disable=unused-argument
+    """Clears future enrollable course runs that were added by this script"""
+    return CourseRun.objects.filter(course=course, edx_course_key__startswith=NEW_COURSE_RUN_PREFIX).delete()
+
+
+@accepts_or_calculates_now
+def add_past_failed_run(user=None, course=None, now=None, grade=DEFAULT_FAILED_GRADE, add_if_exists=False):
+    """Adds a past failed course run for a given user and course"""
+    chosen_past_run = None
+    past_runs = CourseRun.objects.filter(
+        course=course,
+        end_date__lt=now,
+    ).exclude(Q(end_date=None)).order_by('-end_date').all()
+    # Loop through past course runs and find one without CachedCurrentGrade data
+    for past_run in past_runs:
+        if not CachedCurrentGradeHandler(user).get_or_create(past_run).data:
+            chosen_past_run = past_run
+            continue
+        elif not add_if_exists:
+            print("Past failed course run already exists (id: {}, title: {})".format(past_run.id, past_run.title))
```

Comment:
We need to use `stdout.write` here, but also I feel like this should be an `Exception` instead, so that the program return value is non-zero